mappings/net/minecraft/server/player/PlayerManager.mapping

```diff
@@ -0,0 +1,92 @@
+CLASS net/minecraft/class_166 net/minecraft/server/player/PlayerManager
+	FIELD field_577 logger Ljava/util/logging/Logger;
+	FIELD field_578 players Ljava/util/List;
+	FIELD field_579 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_580 chunkTrackers [Lnet/minecraft/class_167;
+	FIELD field_581 maxPlayerCount I
+	FIELD field_582 bannedPlayers Ljava/util/Set;
+	FIELD field_583 ipBans Ljava/util/Set;
+	FIELD field_584 operators Ljava/util/Set;
+	FIELD field_585 whitelist Ljava/util/Set;
+	FIELD field_586 bannedPlayersFile Ljava/io/File;
+	FIELD field_587 bannedIPsFile Ljava/io/File;
+	FIELD field_588 operatorsFile Ljava/io/File;
+	FIELD field_589 whiteListFile Ljava/io/File;
+	FIELD field_590 playerDataLoader Lnet/minecraft/class_168;
+	FIELD field_591 whitelistEnabled Z
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD method_549 getViewRadiusInTiles ()I
+	METHOD method_550 sendPacketToPlayersAround (DDDDILnet/minecraft/class_169;)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 distance
+		ARG 9 dimensionId
+		ARG 10 acketclass_169
+	METHOD method_551 getChunkTracker (I)Lnet/minecraft/class_167;
+		ARG 1 dimension
+	METHOD method_554 (Lnet/minecraft/class_69;)V
+		ARG 1 player
+	METHOD method_556 sendLevelUpdate (Lnet/minecraft/class_69;Lnet/minecraft/class_73;)V
+		ARG 1 player
+		ARG 2 level
+	METHOD method_557 connectPlayer (Lnet/minecraft/class_10;Ljava/lang/String;)Lnet/minecraft/class_69;
+		ARG 1 connect
+		ARG 2 name
+	METHOD method_558 sendPacketToPlayersAround (Lnet/minecraft/class_54;DDDDILnet/minecraft/class_169;)V
```

Comment:
surely surely SUrely you can think of a shorter name

Reply:
no u tell pls

Reply:
distributePacketInRadius idk